### ISSUE TITLE
Inconsistent title colors between the SDK, web beacon and HelpScout Android app.

### ISSUE THREAD

**Issue by sipersso** [helpscout/beacon-android-sdk-sample#111]
**Beacon version:** 2.1.0

1. Open the Android Beacon your app. Notice the color of the title bar and the Answers/Ask tabs (black)
2: Now open the HelpScout Android app. Press your account and then get Help. Notice that the color of the title is white and the answers/ask tabs (white). 
3: Now open a web beacon for your app. Notice the color of the title bar and the Answers/Ask tabs (white)

**Expected:**
It should be possible to match the appearance of the web beacon or the HelpScout Android app. Either the text should be white like the others to be consistent or it should be possible to customize the color. 

**Actual:**
The color of the titlebar is black, which is inconsistent with the web beacon and the HelpScout Android app. And when reading the documentation at https://developer.helpscout.com/beacon-2/android/, there doesn't seem to be a way to customize the ask/tabs text color. 

**My App**
![android](https://user-images.githubusercontent.com/6479433/87249198-e5862d80-c45d-11ea-90fc-f0cf667f0114.png)

**The HelpScout App**
![helpscout_app](https://user-images.githubusercontent.com/6479433/87249234-28480580-c45e-11ea-8860-8cb9a7c02da7.png)

**The Web Beacon**
<img width="391" alt="Skärmavbild 2020-07-12 kl  16 22 52" src="https://user-images.githubusercontent.com/6479433/87249251-4ca3e200-c45e-11ea-8e69-fac09b00ed50.png">

**Comment by scottyab** [comment]
Hey @sipersso the title bar text/icon colors will dynamically switch from light/dark depending on the Beacon color based on the luminance of the color. The lighter colored text/icon in our Help Scout Android app is correct for our brand color. 

Using a colour picker I've sampled the blue color you're using on the Android screenshot as #6fd4e9 is this correct and is it the same hex value you are using on the Web? as I sampled the color in your web screenshot as #47b9c9 

Here's screen shots below the web beacon. 

![Image 2020-07-13 at 11 37 44 am](https://user-images.githubusercontent.com/404105/87296198-4d8c5080-c4fe-11ea-8107-ace00e28db73.png)

![Image 2020-07-13 at 11 47 41 am](https://user-images.githubusercontent.com/404105/87296578-e0c58600-c4fe-11ea-9059-e3d43e0235fc.png)

Let me know when you switch to using the same hex value in both Android and web you're seeing inconsistencies and we can reopen this issue. 

Thanks, 
Scott
